lib/Listener/ShareLinkAccessedListener.php

```diff
@@ -76,6 +76,8 @@ public function handle(Event $event): void {
 			$this->mapper->update($shareLimit);
 		} catch (DoesNotExistException $e) {
 			// No limit is set, ignore
+		} catch (\Exception $e) {
+			// Something went wrong, ignore
```

Comment:
Log it maybe if we do not know how to explain it? Or add an explicit check for missing token earlier and ignore it silently then. I’m always scared by catchall blocks with no log.

Reply:
Done